src/components/page-active-component/active-component-overview.js

```diff
@@ -120,12 +122,27 @@ export class ActiveComponentOverview extends React.Component {
           >
             {component && (
               <React.Fragment>
+                <Toolbar
+                  appName={appName}
+                  envName={envName}
+                  component={component}
+                />
                 <div className="o-layout-columns">
                   <section>
                     <h2 className="o-heading-section">Overview</h2>
                     <p>
                       Component <strong>{component.name}</strong>
                     </p>
+                    {component.status === 'Stopped' && (
+                      <Alert>
+                        Component has been manually stopped; please note that a
+                        new deployment will cause it to be restarted unless you
+                        set replica of the component to 0 in Radix config
```

Comment:
Typo: `replicas`, not `replica`.

Keywords and values should be enclosed in `<code></code>` for formatting, and it would be nice to include a link to the docs here.

Eg:
```…set <code>replicas</code> of the component to <code>0</code> in <a href="https://www.radix.equinor.com/docs/reference-radix-config/#replicas">radixconfig.yaml</a>…```

Reply:
Done